code/modules/mob/living/carbon/human/species_types/IPC.dm

```diff
@@ -148,7 +149,6 @@ datum/species/ipc/on_species_loss(mob/living/carbon/C)
 		if(H.nutrition > NUTRITION_LEVEL_MOSTLY_FULL)
 			to_chat(H, "<span class='notice'>You are now fully charged.</span>")
 			break
-		A.charging = 1
```

Comment:
What's this for?

Reply:
If om not mistaken this just turns on apc charging which is slightly unintended